source/features/extend-profile-nav.tsx

```diff
@@ -0,0 +1,131 @@
+import './extend-profile-nav.css';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import CodeSquareIcon from 'octicon/code-square.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getCleanPathname} from '../github-helpers';
+
+interface UserCounts {
+	repositories: number;
+	projects: number;
+	packages: number;
+	gists: number;
+}
+
+const getUserCounts = cache.function(async (username: string): Promise<UserCounts> => {
+	const {search, user} = await api.v4(`
+		search(type: REPOSITORY query: "user:${username} archived:false") {
+			repositoryCount
+		}
+		user(login: "${username}") {
+			projects(states: OPEN) {
+				totalCount
+			}
+			packages {
+				totalCount
+			}
+			gists {
+				totalCount
+			}
+		}
+	`);
+	return {
+		repositories: search.repositoryCount,
+		projects: user.projects.totalCount,
+		packages: user.packages.totalCount,
+		gists: user.gists.totalCount
+	};
+}, {
+	cacheKey: ([username]) => 'user-counts:' + username
+});
+
+const getOrganizationPackageCount = cache.function(async (organizationName: string): Promise<number> => {
+	const {organization} = await api.v4(`
+	  organization(login: "${organizationName}") {
```

Comment:
Can you merge this into the other query? `remove-projects-tab` used to query both users and organizations in one v4 call; if one failed, it was just `undefined` but it kept working.